common/scripts/x86_64/Ubuntu_16.04/_helpers.sh

```diff
@@ -78,37 +78,48 @@ __check_dependencies() {
   sudo chown -R shippable:shippable /home/shippable/
 
   ################## Install Docker  #####################################
-  __process_msg "Installing Docker $DOCKER_VERSION"
-  rm -f installDockerScript.sh
-  touch installDockerScript.sh
-  echo '#!/bin/bash' >> installDockerScript.sh
-  echo 'install_docker_only="true"' >> installDockerScript.sh
-  echo "SHIPPABLE_HTTP_PROXY=\"$SHIPPABLE_HTTP_PROXY\"" >> installDockerScript.sh
-  echo "SHIPPABLE_HTTPS_PROXY=\"$SHIPPABLE_HTTPS_PROXY\"" >> installDockerScript.sh
-  echo "SHIPPABLE_NO_PROXY=\"$SHIPPABLE_NO_PROXY\"" >> installDockerScript.sh
-
-  local node_scripts_location=/tmp/node
-  local node_s3_location="https://s3.amazonaws.com/shippable-artifacts/node/$RELEASE/node-$RELEASE.tar.gz"
-  pushd /tmp
-  mkdir -p $node_scripts_location
-  wget $node_s3_location
-  tar -xzf node-$RELEASE.tar.gz -C $node_scripts_location --strip-components=1
-  rm -rf node-$RELEASE.tar.gz
-  popd
-
-  cat $node_scripts_location/lib/logger.sh >> installDockerScript.sh
-  cat $node_scripts_location/lib/headers.sh >> installDockerScript.sh
-  cat $node_scripts_location/initScripts/$ARCHITECTURE/$OPERATING_SYSTEM/Docker_$DOCKER_VERSION.sh >> installDockerScript.sh
-
-  rm -rf $node_scripts_location
-  # Install Docker
-  chmod +x installDockerScript.sh
-  ./installDockerScript.sh
-
-  # sets env INSTALLED_DOCKER_VERSION
-  __set_installed_docker_version
-
-  rm installDockerScript.sh
+  {
```

Comment:
this can just be
`check_docker=$(sudo service --status-all 2>&1 | grep docker) || true`

Reply:
then it'll be always true wont it?

Reply:
no it will be true if the command `sudo service --status-all 2>&1 | grep docker` fails

Reply:
this is incorrect. Merging this based on the conversation on slack with @ric03uec  that this is done according to the conventions followed previously in admiral.